packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -530,9 +539,23 @@ class FlutterLocalFileComparator extends FlutterGoldenFileComparator with LocalC
     }
 
     for (MapEntry<String, ComparisonResult> entry in failureDiffs.entries) {
-      if (await skiaClient.isValidDigestForExpectation(entry.key, golden.path))
-        generateFailureOutput(entry.value, golden, basedir, key: entry.key);
+      generateFailureOutput(entry.value, golden, basedir, key: entry.key);
     }
     return false;
   }
 }
+
+Future<List<String>> _validTestExpectations(
```

Comment:
This may not go over well. 

The reason why expectation parameters are _only_ validated after a match is not found is because the Gold `/details` api is expensive and subject to a rate limit. The request may timeout if you check the `/details` like this for _every_ test and _every_ expectation. It would also mean a lot of unnecessary requests.

If there is a matching image from the given expectations, then checking the `/details` is irrelevant, so they are checked only when a match cannot be found in the current implementation.

cc/ @kjlubick

Reply:
Yeah, check (and cache) the baseline from /json/expectations, since that will have all known digests for all tests and is very cheap (much higher rate limit).

Reply:
As mentioned in another comment, we are going to wait for TryJobs for this, reverted the changes here.